google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/it/BigtableBackupIT.java

```diff
@@ -38,111 +37,72 @@
 import com.google.cloud.bigtable.admin.v2.models.Table;
 import com.google.cloud.bigtable.admin.v2.models.UpdateBackupRequest;
 import com.google.cloud.bigtable.data.v2.BigtableDataClient;
-import com.google.cloud.bigtable.data.v2.models.RowMutation;
-import com.google.cloud.bigtable.test_helpers.env.AbstractTestEnv;
+import com.google.cloud.bigtable.data.v2.models.RowMutationEntry;
 import com.google.cloud.bigtable.test_helpers.env.EmulatorEnv;
 import com.google.cloud.bigtable.test_helpers.env.TestEnvRule;
-import com.google.common.collect.Lists;
 import com.google.protobuf.ByteString;
 import io.grpc.StatusRuntimeException;
 import java.io.IOException;
 import java.util.List;
 import java.util.Random;
 import java.util.concurrent.ExecutionException;
-import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
 import java.util.logging.Logger;
-import org.junit.*;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 import org.threeten.bp.Duration;
 import org.threeten.bp.Instant;
 
 @RunWith(JUnit4.class)
 public class BigtableBackupIT {
-  @ClassRule public static TestEnvRule testEnvRule = new TestEnvRule();
+  @ClassRule public static final TestEnvRule testEnvRule = new TestEnvRule();
 
   private static final Logger LOGGER = Logger.getLogger(BigtableBackupIT.class.getName());
 
   private static final int[] BACKOFF_DURATION = {2, 4, 8, 16, 32, 64, 128, 256, 512, 1024};
 
-  private static final String TEST_TABLE_SUFFIX = "test-table-for-backup-it";
-  private static final String TEST_BACKUP_SUFFIX = "test-backup-for-backup-it";
-
   private static BigtableTableAdminClient tableAdmin;
   private static BigtableInstanceAdminClient instanceAdmin;
   private static BigtableDataClient dataClient;
 
-  private static String targetInstance;
   private static String targetCluster;
   private static Table testTable;
-  private static String prefix;
 
   @BeforeClass
-  public static void createClient()
-      throws IOException, InterruptedException, ExecutionException, TimeoutException {
+  public static void setUpClass() throws InterruptedException {
     assume()
         .withMessage("BigtableInstanceAdminClient is not supported on Emulator")
         .that(testEnvRule.env())
         .isNotInstanceOf(EmulatorEnv.class);
 
+    tableAdmin = testEnvRule.env().getTableAdminClient();
```

Comment:
It shouldnt cause flakes since all of the names are unique now. I'm actually hoping to limit the flakiness by reducing the resource churn in this test.